### PR TITLE
feat: add single sided crv disclosure

### PR DESCRIPTION
**Pull request by banteg**

### CODE REVIEW

File: disclosures/2021-05-14.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Incident disclosure 2021-05-14
+
+## Summary
+- An conservative setting on `SingleSidedCrvDAI` strategy has led to the strategy recording a large on paper loss when it was harvested out of turn.
```
Comment (Grandthrax) [inline disclosures/2021-05-14.md:4]:
```suggestion
- A conservative setting on `SingleSidedCrvDAI` strategy has led to the strategy recording a large on paper loss when it was harvested out of turn.
```

File: disclosures/2021-05-14.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The first `harvest` of the fixed strategy has offset the paper loss and reported a 5,332,803 DAI profit, which is well in excess of the loss reported in the incident harvest[[3]](#References).
+
+## Timeline of events
+1. **May 14, 2021 08:32 (UTC)** A keeper calls harvest[[1]](#References) on `SingleSidedCrvDAI`, meaning to harvest `SingleSidedCrvUSDC`. This causes a loss to be reported and the share price of yvDAI to decrease.
+3. **09:42** A user reports an unusual drop in yvDAI vault share price from 1.04323
+ to 1.03531.
+5. **10:13** Root cause is identified and the fix is produced[[2]](#References). 
+6. **10:49** A fixed `SingleSidedCrvDAI` is deployed[[3]](#References) and the migration is queued in Yearn multisig.
+7. **11:39** Migrate transaction is executed[[3]](#References) and the share price returns to normal.
```
Comment (Grandthrax) [inline disclosures/2021-05-14.md:34]:
```suggestion
7. **11:39** Migrate transaction is executed[[4]](#References) and the share price returns to normal.
```

File: disclosures/2021-05-14.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Incident disclosure 2021-05-14
+
+## Summary
+- A method `SingleSidedCrvDAI` uses to estimate its assets has led the strategy to believe it had suffered losses.
```
Comment (milkyklim) [inline disclosures/2021-05-14.md:4]: uses => used?

File: disclosures/2021-05-14.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The value of the strategy's position is calculated in the `estimatedTotalAssets` function. It aims to return the total value that could *actually* be obtained from the strategy if it were to divest its entire position based on the current on-chain conditions.
+
+It does this by simulating a withdrawal of all assets from the Curve pool back to DAI. As the strategy grew over time the size of this simulation increased until it was simulating a single 85,270,663 DAI withdrawal. Behind the scenes this is equivalent of selling a very large amount of USDT and USDC in a single trade which would cause a large loss.
+
+## Details of Incident
+
+A loss of 4,932,713 DAI was incorrectly reported during the `harvest` function call[[1]](#References) which updates the vault's profit and loss figures. This amount is equivalent to the slippage of a single-sided 85.27 million DAI withdrawal.
+
+This was a paper loss as the trade doesn't happen. It was a simulation to determine how much DAI would be returned if all funds were divested at once.
```
Comment (milkyklim) [inline disclosures/2021-05-14.md:20]: doesn't => didn't

**Review by lehnberg — APPROVED** [review]
Nice updates, looks good to me, LGTM

**Pull request merged**